src/uimodel.cpp

```diff
@@ -2144,27 +2144,33 @@ bool UiModel::Process()
   return m_Running;
 }
 
+bool UiModel::CompareChats(const ChatInfo& lhsChatInfo, const ChatInfo& rhsChatInfo)
```

Comment:
maybe should rename this params with p_ prefix

Reply:
Yes, that would be neat 👍